urlman.py

```diff
@@ -31,10 +31,12 @@ class UrlsMetaclass(type):
     """
     def __new__(self, name, bases, attrs):
         # Collect patterns off of the attrs
+        callables = [
```

Comment:
I'd probably move this into a class variable rather than initialising it every time.

Reply:
That makes more sense, definitely. Pushed :)